skyportal/handlers/api/analysis.py

```diff
@@ -915,6 +946,18 @@ def get(self, analysis_resource_type, analysis_id=None):
                   schema: Error
         """
         if analysis_resource_type.lower() == 'obj':
+            include_analysis_data = self.get_query_argument(
+                "includeAnalysisData", False
```

Comment:
Should this query be checked outside the if statement?

Reply:
Yes, since it can be used for other `analysis_resource_type`s in the future.